ERROR - 2022-11-29 22:12:35 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'pixel', 3306, '', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Home.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Home->index()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\test\da_sein\app\Controllers\Home.php(18): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Home->index()
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-11-29 22:12:35 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `slider`', [], false)
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
